modules/mutect2/1.0/mutect2.smk

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env snakemake
```

Comment:
I have one last request. Per #70, could you please change all instances of 
```
resources: 
    mem_mb = CFG["rulename"]["mem_mb"]
```

to the new unpacking style? It lets users feed in additional resource parameters.